src/ai/backend/client/func/session.py

```diff
@@ -218,7 +218,9 @@ async def get_or_create(
             access key.
         :param mount_map: Mapping which contains custom path to mount vfolder.
             Key and value of this map should be vfolder name and custom path.
-            All custom mounts should be under /home/work.
+            Defalut mounts or relative paths are under /home/work.
+            If you want different paths, names should be absolute paths.
+            vFolders should have different name from linux system folders.
```

Comment:
```suggestion
            The target mount path of vFolders should not overlap with the linux system folders.
```

vFolder name != target mount path.

Please update other places as well.